Makefile

```diff
@@ -241,67 +172,3 @@ endif
-
-.PHONY: server-debug-deploy
-server-debug-deploy: validate-go-version
-	./build/bin/manifest apply
-	mkdir -p server/dist
-ifeq ($(OS),Darwin)
-	cd server && env GOOS=darwin GOARCH=amd64 $(GOBUILD) -gcflags "all=-N -l" -o dist/plugin-darwin-amd64;
-else ifeq ($(OS),Linux)
-	cd server && env GOOS=linux GOARCH=amd64 $(GOBUILD) -gcflags "all=-N -l" -o dist/plugin-linux-amd64;
-else ifeq ($(OS),Windows_NT)
-	cd server && env GOOS=windows GOARCH=amd64 $(GOBUILD) -gcflags "all=-N -l" -o dist/plugin-windows-amd64.exe;
-else
-	$(error make debug depends on uname to return your OS. If it does not return 'Darwin' (meaning OSX), 'Linux', or 'Windows_NT' (all recent versions of Windows), you will need to edit the Makefile for your own OS.)
-endif
-	rm -rf dist/
-	mkdir -p dist/$(PLUGIN_ID)/server/dist
-	cp $(MANIFEST_FILE) dist/$(PLUGIN_ID)/
-	cp -r server/dist/* dist/$(PLUGIN_ID)/server/dist/
-	mkdir -p ../mattermost-server/plugins
-	cp -r dist/* ../mattermost-server/plugins/
-
-.PHONY: validate-go-version
-validate-go-version: ## Validates the installed version of go against Mattermost's minimum requirement.
-	@if [ $(GO_MAJOR_VERSION) -gt $(MINIMUM_SUPPORTED_GO_MAJOR_VERSION) ]; then \
-		exit 0 ;\
-	elif [ $(GO_MAJOR_VERSION) -lt $(MINIMUM_SUPPORTED_GO_MAJOR_VERSION) ]; then \
-		echo '$(GO_VERSION_VALIDATION_ERR_MSG)';\
-		exit 1; \
-	elif [ $(GO_MINOR_VERSION) -lt $(MINIMUM_SUPPORTED_GO_MINOR_VERSION) ] ; then \
-		echo '$(GO_VERSION_VALIDATION_ERR_MSG)';\
-		exit 1; \
-	fi
-
-# Reset the plugin
-.PHONY: reset
-reset:
-ifeq ($(and $(MM_SERVICESETTINGS_SITEURL),$(MM_ADMIN_USERNAME),$(MM_ADMIN_PASSWORD),$(CURL)),)
-	$(error In order to use make reset, the following environment variables need to be defined: MM_SERVICESETTINGS_SITEURL, MM_ADMIN_USERNAME, MM_ADMIN_PASSWORD, and you need to have curl installed.)
-endif
-
-# If we were debugging, we have to unattach the delve process or else we can't disable the plugin.
-# NOTE: we are assuming the dlv was listening on port 2346, as in the debug-plugin.sh script.
-	@DELVE_PID=$(shell ps aux | grep "dlv attach.*2346" | grep -v "grep" | awk -F " " '{print $$2}') && \
-	if [ "$$DELVE_PID" -gt 0 ] > /dev/null 2>&1 ; then \
-		echo "Located existing delve process running with PID: $$DELVE_PID. Killing." ; \
-		kill -9 $$DELVE_PID ; \
-	fi
-
-	@echo "\nRestarting plugin via API"
-	$(eval TOKEN := $(shell curl -i -X POST $(MM_SERVICESETTINGS_SITEURL)/api/v4/users/login -d '{"login_id": "$(MM_ADMIN_USERNAME)", "password": "$(MM_ADMIN_PASSWORD)"}' | grep -o "Token: [0-9a-z]*" | cut -f2 -d' ' 2> /dev/null))
-	@curl -s -H "Authorization: Bearer $(TOKEN)" -X POST $(MM_SERVICESETTINGS_SITEURL)/api/v4/plugins/$(PLUGIN_ID)/disable > /dev/null && \
-		curl -s -H "Authorization: Bearer $(TOKEN)" -X POST $(MM_SERVICESETTINGS_SITEURL)/api/v4/plugins/$(PLUGIN_ID)/enable > /dev/null && \
-		echo "OK." || echo "Sorry, something went wrong. Check that MM_ADMIN_USERNAME and MM_ADMIN_PASSWORD env variables are set correctly."
-
-
```

Comment:
Please leave these targets in the file. I'm aware that it's consistent with that is in the ticket, but it's not worth the cleanup in this PR.

Reply:
Gotcha.